core/src/mysql/federation.rs

```diff
@@ -29,9 +31,9 @@ impl MySQLTable {
         let fed_provider = Arc::new(SQLFederationProvider::new(self));
         Ok(Arc::new(SQLTableSource::new_with_schema(
             fed_provider,
-            table_name,
+            RemoteTableRef::try_from(table_name)?,
```

Comment:
Instead of using try_from here, RemoteTableRef can be directly created from table reference
```rust
let remote_table_ref = RemoteTableRef::from(self.base_table.table_reference.clone());
```

Reply:
Fixed